### DISCUSSION TITLE
Display top discussions

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [giscus/giscus#520] [Ideas]
If giscus is added to multiple pages, it would be nice to show on the main index page the top / recent discussions, so people can be directed to the pages where active discussions are taking place.

**Comment by laymonage** [comment]
The Discussions API only supports ordering by [`CREATED_AT` and `UPDATED_AT`](https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions#repositorydiscussions), so this isn't possible yet.

**Reply by AustenLamacraft** [reply]
Ok but for small sites CREATED_AT may be enough.
